src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/ImportFAT2SectorStandardToPL.sql

```diff
@@ -0,0 +1,137 @@
+﻿
+CREATE PROCEDURE [dbo].[ImportFAT2SectorStandardToPL]
+(
+   @RunId int
+)
+AS
+-- ==========================================================================================================
+-- Author:      Sarma Evani
+-- Create Date: 17/Dec/2020
+-- Description: Import, Transform and Load FAT2 Framework Presentation Layer Table
+-- ==========================================================================================================
+BEGIN TRY
+		DECLARE @LogID int
+
+		/* Start Logging Execution */
+
+			  INSERT INTO Mgmt.Log_Execution_Results
+				  (
+					RunId
+				   ,StepNo
+				   ,StoredProcedureName
+				   ,StartDateTime
+				   ,Execution_Status
+				  )
+			  SELECT 
+					@RunId
+				   ,'Step-6'
+				   ,'ImportFAT2SectorStandardToPL'
+				   ,getdate()
+				   ,0
+
+			  SELECT @LogID=MAX(LogId) FROM Mgmt.Log_Execution_Results
+			  WHERE StoredProcedureName='ImportFAT2SectorStandardToPL'
+			  AND RunId=@RunID
+
+			BEGIN TRANSACTION
+
+					DELETE FROM [ASData_PL].[FAT2_StandardSector]
+
+					INSERT [ASData_PL].[FAT2_StandardSector]
+					(												
+						[Id],
+						[Title],
+						[Level],
+						[IntegratedDegree],
+						[OverviewOfRole],
+						[ApprenticeshipStandardId],
```

Comment:
Can you call this VaApprenticeshipStandardId instead please?

Reply:
Changed the column to VaApprenticeshipStandardId